File: Finah-Desktop-Java/src/be/pxl/objects/AnswerList.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package be.pxl.objects;


public class AnswerList {
	private int id;
	private int time;
	private String date;
	private int list;
	private int answer;
	private int question;
	private int workpoint;
	private String hash;
	private int usertype;
	
	
	
	public AnswerList(int id, int time, String date, int list, int answer, int question,
			int workpoint, String hash, int usertype) {
	
		this.id = id;
		this.list = list;
		this.answer = answer;
		this.question = question;
		this.workpoint = workpoint;
		this.hash = hash;
		this.date = date;
		this.usertype = usertype;
		this.time = time;
	}
	
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public int getList() {
		return list;
	}
	public void setList(int list) {
		this.list = list;
	}
	public int getAnswer() {
		return answer;
	}
	public void setAnswer(int answer) {
		this.answer = answer;
	}
	public int getQuestion() {
		return question;
	}
	public void setQuestion(int question) {
		this.question = question;
	}
	public int getWorkpoint() {
		return workpoint;
	}
	public void setWorkpoint(int workpoint) {
		this.workpoint = workpoint;
	}
	public String getHash() {
		return hash;
	}
	public void setHash(String hash) {
		this.hash = hash;
	}
	public String getDate() {
		return date;
	}
	public void setDate(String date) {
		this.date = date;
	}
	public int getUsertype() {
		return usertype;
	}
	public void setUsertype(int usertype) {
		this.usertype = usertype;
	}
	public int getTime() {
		return time;
	}
	public void setTime(int time) {
		this.time = time;
	}
	

	

	
}

File: Finah-Desktop/Finah-DataObjects/AnswerList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataObjects
{
    public class AnswerList
    {
        public AnswerList()
        {

        }

        public int Id { get; set; }
        public int List { get; set; }
        public int Answer { get; set; }
        public int Question { get; set; }
        public int Workpoint { get; set; }
        public string Hash { get; set; }
        public DateTime Date { get; set; }
        public int UserType { get; set; }
        public int Time { get; set; }


        public void SetAnswerList(int id, int list, int answer, int question, int workpoint, string hash, DateTime date, int userType, int Time)
        {
            this.Id = id;
            this.List = list;
            this.Answer = answer;
            this.Question = question;
            this.Workpoint = workpoint;
            this.Hash = hash;
            this.Date = date;
            this.UserType = userType;
            this.Time = Time;


        }
    }
}
